src/field/pokemon.ts

```diff
@@ -3504,12 +3504,16 @@ export abstract class Pokemon extends Phaser.GameObjects.Container {
     return this.isBoss();
   }
 
-  isMax(): boolean {
+  /**
+   * @param includeEternamax Whether or not to include Eternamax
+   * @returns if the Pokemon is in a max form
+   */
+  isMax(includeEternamax: boolean = true): boolean {
     const maxForms = [
       SpeciesFormKey.GIGANTAMAX,
       SpeciesFormKey.GIGANTAMAX_RAPID,
       SpeciesFormKey.GIGANTAMAX_SINGLE,
-      SpeciesFormKey.ETERNAMAX,
+      ...(includeEternamax ? SpeciesFormKey.ETERNAMAX : []),
```

Comment:
this should be an array
```suggestion
      ...(includeEternamax ? [SpeciesFormKey.ETERNAMAX] : []),
```